README.md

```diff
@@ -9,7 +9,7 @@ This repository contains new components meant to be used in conjunction with the
 
 ## Setup
 
-Requires Node JS >= 8.0.
+Requires Node JS >= 10.13.0.
```

Comment:
Out of curiosity, why 10.13.0 specifically? The latest 10.x version is 10.21.0.

Reply:
Bad google cache, I was checking the snippet from Node JS. Will update. Imposing Node 12.18.2, because it's the current LTS.

I'm a fervent partisan to try to always be on the current LTS, and not the maintenance LTS, just because its a good way to prevent late update.